### PR TITLE
multi arch build error

### PR DESCRIPTION
**Pull request by lxm**
Signed-off-by: lxm <lxm.xupt@gmail.com>

fix multi arch build error

### CODE REVIEW

File: hack/docker_build.sh [hunk 1]
```diff
@@ -40,7 +40,7 @@ if [[ $? != 0 ]]; then
   echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
 fi
 
-TAG=$TAG-multiarch
+TAG=$TAG
```
Comment (LinuxSuRen) [inline hack/docker_build.sh:43]: I think this line should be removed.

`TAG=$TAG` has no meaning.

**Review by LinuxSuRen — CHANGES_REQUESTED** [review]
Thanks for the fixing. Please check out my comments.

By the way, I don't understand the following code line in the multi-arch build GitHub action:

https://github.com/kubesphere/kubesphere/blob/cad6296f4e27765ca58b931e2c3a4afb4af6a963/.github/workflows/build-multiarch.yaml#L8

```
on:
  push:
    branches:
      - 'master'
      - 'release*'
      - '*'
```

The wildcard `*` means all branches. I would suggest removing the wildcard. Or please provide a reasonable description.

**Comment by lxm** [comment]
> Thanks for the fixing. Please check out my comments.
> 
> By the way, I don't understand the following code line in the multi-arch build GitHub action:
> 
> https://github.com/kubesphere/kubesphere/blob/cad6296f4e27765ca58b931e2c3a4afb4af6a963/.github/workflows/build-multiarch.yaml#L8
> 
> ```
> on:
>   push:
>     branches:
>       - 'master'
>       - 'release*'
>       - '*'
> ```
> 
> The wildcard `*` means all branches. I would suggest removing the wildcard. Or please provide a reasonable description.

is was used to test in my own repo, now remove it

File: hack/docker_build.sh (hunk 1 above)
Comment (lxm) [reply hack/docker_build.sh:43]: right, removed this line . it was unexpected to add to this file

**Review by LinuxSuRen — APPROVED** [review]
/lgtm

**Pull request merged**